### ISSUE TITLE
Mandatory, Recommended and Optional properties for ConceptScheme and Concept

### ISSUE THREAD

**Issue by ChLaaboudi** [linked-statistics/xkos#179]
In G.1 **Description of statistical classifications**, in the table (paragraph 5.1) the property <skos:hasTopConcept> is missing,

Would it be possible to add a table summarising the properties (mandatory, recommended, optional)  for  the Classifications items (G.5 Classification items)

**Comment by tfrancart** [comment]
Hello @ChLaaboudi 

> In G.1 Description of statistical classifications, in the table (paragraph 5.1) the property skos:hasTopConcept is missing,

There is already a pending pull request for this (https://github.com/linked-statistics/xkos/pull/175), waiting for the review of @FranckCo for merge.

> Would it be possible to add a table summarising the properties (mandatory, recommended, optional) for the Classifications items (G.5 Classification items)

I don't know, maybe this would be misinterpreted as a too restrictive list of properties by readers ? I feel this is more open that the statistical classifications themselves. @FranckCo what do you think ?

**Comment by FranckCo** [comment]
I agree that adding a table of properties for classification items is useful. This needs a bit of thinking, though, so it will be done with the future version of XKOS.
